test/main/signers/hot/RingSigner/index.test.js

```diff
@@ -43,7 +43,9 @@ describe('Ring signer', () => {
 
   afterAll(() => {
     clean()
-
+    if (signer.status !== 'locked') {
```

Comment:
after all the tests are run we should know what state the signer is in. what's this check for?